TACC/Core-Portal
# Task/FP-339: Search backend enhancements

## Overview: ##
Changes/enhancements to the search backend.

## PR Status: ##


## Related Jira tickets: ##

* [FP-339](https://jira.tacc.utexas.edu/browse/FP-339)

## Summary of Changes: ##
- Indexing tasks now use bulk operations where possible
- Indexed file UUIDs are now the SHA256 hash of the system and path, so files can be retrieved/updated by UUID without doing a separate lookup search.
- Remove unnecessary listings in `walk_levels`
## Testing Steps: ##
1. `from portal.libs.elasticsearch.indexes import setup_files_index`
2. `setup_files_index()`
3. Perform listings in Data Files and curl `http://cep.dev:9200/frontera-dev-files/_search` to check that indexing has occurred.

## Notes: ##
